ironfish/src/testUtilities/helpers/blockchain.ts

```diff
@@ -36,11 +43,25 @@ export async function makeBlockAfter(
   const randomness = BigInt(Math.floor(Math.random() * Number.MAX_SAFE_INTEGER))
   const graffiti = GraffitiUtils.fromString('fake block')
 
+  const key = generateKey()
+  const owner = key.public_address
+  const minerNote = new NativeNote(owner, BigInt(0), '', Asset.nativeIdentifier(), owner)
+  const transaction = new NativeTransaction(key.spending_key)
+  transaction.receive(minerNote)
+  const minerTransaction = new Transaction(transaction.post_miners_fee())
+
+  const noteCommitment = await chain.db.withTransaction(null, async (tx) => {
+    await chain.notes.addBatch(minerTransaction.notes, tx)
+    const commitment = await chain.notes.rootHash(tx)
+    await tx.abort()
```

Comment:
Not sure this will work unless we have all the previous miners fee transactions in note tree as well. This just adds the current one right? Definitely think we need to remove this `makeBlockAfter` function or find some way of making a long string of blocks without needing to update the DB every time (like in memory merkle tree for example). Also `makeBlockAfter` is used in a bunch of places so hard to get rid of